config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -197,9 +197,9 @@ choice OPENTHREAD_SECURITY
 	default OPENTHREAD_NRF_SECURITY_CHOICE
 endchoice
 
-config CC3XX_BACKEND
-    bool
-    default n
+config PSA_CRYPTO_DRIVER_CC3XX
+	bool
+	default n
 
 config OBERON_BACKEND
```

Comment:
Not that needs to be set by this. The issue we are solving is to ensure that legacy configuration is still viable for CHIP which (for now) defaults to nrf_oberon because of size requirements